source/conf.py

```diff
@@ -47,6 +47,9 @@
 # Disable following anchors in URLS for linkcheck
 linkcheck_anchors = False
 
+# Linkcheck Exclusions
+linkcheck_ignore = [r'.*kauailabs.com.*']
```

Comment:
This will ignore anything with `kauailabs.com` in the address. Is that what we want to do or do we want to be more selective?

Reply:
That's probably more open then it needs to be, but I think it's also not going to cause a problem. It's highly unlikely that someone else makes a domain name that contains kauailabs.com that we would reference in the docs.